readabilipy/simple_json.py

```diff
@@ -44,11 +48,12 @@ def simple_json_from_html_string(html, content_digests=False, node_indexes=False
             f_html.write(html)
             f_html.close()
         html_path = f_html.name
-        
+
         # Call Mozilla's Readability.js Readability.parse() function via node, writing output to a temporary file
         article_json_path = f_html.name + ".json"
         jsdir = os.path.join(os.path.dirname(__file__), 'javascript')
-        subprocess.check_call(["node", "ExtractArticle.js", "-i", html_path, "-o", json_path], cwd=jsdir)
+        subprocess.check_call(
+            ["node", "ExtractArticle.js", "-i", html_path, "-o", article_json_path], cwd=jsdir)
```

Comment:
note that tests fail on master currently because there's an undefined variable `json_path`---this provides a fix.